Editor/Window/GameLiftPlugin.cs

```diff
@@ -119,7 +56,7 @@ private void ApplyText()
             l.SetElementText(Pages.Help, Strings.TabHelp);
         }
 
-        private void OpenTab(string tabName)
+        internal void OpenTab(string tabName)
```

Comment:
Now that we're accessing this method outside the class, we should no longer accept generic strings. Instead we update `Pages` to an enum (with string internal state) and update the `tabName` parameter to that enum instead of string

Reply:
Kept OpenTab(string) as private, created internal OpenTab(Pages) for outside of this class usage